src/containers/checkout/checkout-form/tests/checkout-form.spec.js

```diff
@@ -33,9 +61,19 @@ const userData = {
   snackBarSeverity: '',
   snackBarMessage: ''
 };
+const formik = {
+  values: {},
+  touched: {},
+  errors: {},
+  handleBlur: mockBlur,
+  resetForm: mockReset,
+  handleSubmit: mockSubmit,
+  handleChange: mockChange
+};
 
 useDispatch.mockImplementation(() => dispatch);
 useSelector.mockImplementation(() => userData);
+useFormik.mockImplementation(() => formik);
```

Comment:
We don't need to mock 'formik'. Here are instructions on how to test formik.

https://testing-library.com/docs/example-react-formik/

Reply:
I'm already working on it